packages/expo-dev-menu/app/components/Main.tsx

```diff
@@ -188,6 +202,15 @@ export function Main() {
         </View>
       </View>
 
+      {!hasDisabledDevSettingOption && (
+        <>
+          <Spacer.Vertical size="large" />
+          <Text size="small" color="secondary" align="center">
+            Some development settings are unavailable for this build.
```

Comment:
```suggestion
            Some development settings are unavailable for this development build.
```
I would add `development` to this message to indicate that we're describing the native build of the application. What do you think?

Reply:
love it! thanks